habitat-lab/habitat/tasks/rearrange/multi_task/pddl_sim_state.py

```diff
@@ -213,9 +216,31 @@ def set_state(
             )
             agent_data.articulated_agent.base_pos = start_pos
             agent_data.articulated_agent.base_rot = start_rot
+            # The second attemp should just place the robot at the random location
+            if was_fail:
```

Comment:
Fix the underlying cause of this problem rather than modifying this. Randomly placing the robot in this call could have bad side effects.

Reply:
Thanks, remove this!